hbase-server/src/main/java/org/apache/hadoop/hbase/wal/WALPrettyPrinter.java

```diff
@@ -274,7 +287,8 @@ public void processFile(final Configuration conf, final Path p)
         Map<String, Object> txn = key.toStringMap();
         long writeTime = key.getWriteTime();
         // check output filters
-        if (table != null && !((TableName) txn.get("table")).toString().equals(table)) {
+        if (!tableSet.isEmpty() &&
```

Comment:
nit: I think the second condition covers the first one.

Reply:
I think we need the first one where we don't pass `-t` param and set is Empty. In that case, the second one will always skip all the tables.